statsmodels/tools/sequences.py

```diff
@@ -0,0 +1,185 @@
+"""Low discrepancy sequence tools."""
+import numpy as np
```

Comment:
as general precaution add
`from __future__ import division`
so we don't have to worry about integer division and differences py2 versus py3

Reply:
OK